src/main/java/hudson/tasks/test/AbstractTestResultAction.java

```diff
@@ -70,7 +79,6 @@
 public abstract class AbstractTestResultAction<T extends AbstractTestResultAction> implements HealthReportingAction, RunAction2 {
 
     private static final Logger LOGGER = Logger.getLogger(AbstractTestResultAction.class.getName());
-
```

Comment:
unwanted change

Reply:
I might have mistakenly presses an enter here. Thanks for pointing it out.